### ISSUE TITLE
USWDS - Bug: skipnav text overlays content at high zoom levels

### ISSUE THREAD

**Issue by n1zyy** [uswds/uswds#6260]
### Describe the bug

At high zoom levels, the "Skip to main content" skipnav component's text becomes visible when it should not be, and overlays content. I have attached a screenshot showing this on the USWDS site itself.

Most users will not browse the web this way, but there's a certain cruel irony in this is most likely to affect users with reduced vision and it's an accessibility component that's getting in the way.

This came up as an issue on Login.gov but in debugging I realized it applies to USWDS itself.

### Steps to reproduce the bug

1. Browse a USWDS site in Chrome on an iPhone (this likely applies more broadly)
2. Using the three dots at the bottom of the Chrome UI, select "Zoom text..."
3. Zoom in multiple levels

### Expected Behavior

The skipnav content does not become visible unless triggered, regardless of zoom level.

### Screenshots

![Image](https://github.com/user-attachments/assets/905341b9-8c96-4321-a11b-735fb0765366)
^ Chrome on iPhone (iOS 18.2 on an iPhone 14)

![Image](https://github.com/user-attachments/assets/ccfd1a19-41fd-4a0d-b0d7-783cf4f2976c)
^ same Chrome on iPhone setup, but on login.gov where the issue was first reported

![Image](https://github.com/user-attachments/assets/383c78cf-88a1-4b20-9a54-9ac2409eeab0)
^ a less realistic use case, but Chrome on a Mac (visiting local login.gov dev site)

### System setup

- USWDS version: 3.9.0 locally, but also designsystem.digital.gov itself
- Device: iPhone 14
- OS: iOS 18.2
- Browser: Chrome (131.0.6778.154)

I cannot reproduce it in Safari on iOS, but do not have a variety of devices to test on.

I was only able to reproduce this on a desktop by making the browser window very small and zooming in a ton; it made it easier for me to poke at for debugging, but I suspect it is far less likely to be encountered in the wild this way.

### Additional context

I was experimentally able to fix this by changing the `skipnav` component's `top` attribute in CSS from `-3.8rem` to `-38rem`, but this is likely an inelegant fix. (I am not a designer.)

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by n1zyy** [comment]
Digging into this further, it seems that what is happening is that when the skipnav text wraps to a third line, -3.8rem is no longer enough to keep it out of view.

The Chrome vs. Safari issue was a red herring; it's actually the "Zoom text" feature vs. using gestures. Making the text larger (with Chrome's "Zoom text" selection) keeps the width constrained, so it's much easier for "Skip to main content" to take up three lines. Pinch-zoom in Chrome behaves the same as in Safari and the issue doesn't occur.

Theoretically you could put a paragraph of text in there and make this happen 100% of the time, but I think it's worth trying to prevent real-world content like "Skip to main content" from spilling over content when someone on a smartphone uses very large text. I found `top: -5rem` prevented that from happening at max zoom.

**Comment by aduth** [comment]
One possible solution could include switching from a negative `top` offset to a `bottom: 100%`, to ensure that the bottom of the off-screen content aligns with the top of the page.

Demo: https://codepen.io/aduth/pen/mybWbea

**Comment by amycole501** [comment]
I can confirm this does happen on an iphone 11 in Chrome. I can't reproduce it in Firefox. Seems to be just a Chrome thing.

**Comment by dct2023** [comment]
@amycole501 Do we need to slate this to an upcoming sprint? Or does this need additional details?

**Comment by amycole501** [comment]
I don't have any other details to provide.  It could go into a sprint after the Elements release.

**Comment by amycole501** [comment]
I don't think so but you'll have to check with the dev team.

**Comment by dct2023** [comment]
@amycole501 Thank you for the note! @mejiaj Thoughts on if this is one we need to prioritize?
